### DISCUSSION TITLE
Does it works with os_signpost ?

### DISCUSSION THREAD

**Discussion by mackoj** [dataFude-app/dataFude#14] [Q&A] [answered]
Hi @icanzilb,

Since you support `os_log` do you plan to make it work with `os_signpost` ?

https://www.swiftbysundell.com/wwdc2018/getting-started-with-signposts/

Thanks,

**Comment by icanzilb** [comment, marked as answer]
Yes that's a great question @mackoj — the app already includes the code that tracks signposts I just need a moment to find the most efficient way to consume the log stream because enabling signposts increases the the workload. This is something I'd like to release in the next minor version or the one after that one. I currently have a timer tile that tracks signpost interval duration and in case the same interval repeats - the total duration for all repetitions. 

In case you think of more features that could be useful, let me know here. Thank you!

**Reply by icanzilb** [reply]
Here's a simple screenshot of the wip tile that shows a timer based on os_signpost marks: 
![image](https://user-images.githubusercontent.com/633862/220743277-c106347a-7a17-44de-a0a5-05e22aa8f310.png)

**Reply by mackoj** [reply]
Thanks for taking the time to respond.

Your app is great I'm trying to make the most of it.
